distribution/sdk/sample-content-metadata-extractor/pom.xml

```diff
@@ -60,13 +60,16 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package />
+                        <Export-Package>
```

Comment:
❔ Are we using this SDK module at runtime?

Reply:
No, I believe this was only to get integration tests to pass. The SDK app is used in the tests but not in the profiles.